src/Brouter/Demo/Bit.Brouter.Demo.Core/Shared/MainLayout.razor.css

```
@@ -0,0 +1,6 @@
+.page {
+}
```

Stylelint (17.11.1)

[error] Empty block (block-no-empty)

(block-no-empty)